ユーザー本人確認ページ(マークアップ)

Ken-Yamamoto opened the pull request:
#WHAT
ユーザー情報登録ページの追加
#WHY
必要な機能のため

Ken-Yamamoto committed:
Add userRegistration.html.haml & userRegistration.scss

ユーザー情報登録ページの追加

Ken-Yamamoto commented:
作成分1
[![Image from Gyazo](https://i.gyazo.com/aa6a90b4e13b33484dc271b3bdc4b483.png)](https://gyazo.com/aa6a90b4e13b33484dc271b3bdc4b483)

Ken-Yamamoto commented:
作成分2
[![Image from Gyazo](https://i.gyazo.com/403ec9093ae7d27bbc628e2253712242.png)](https://gyazo.com/403ec9093ae7d27bbc628e2253712242)

Ken-Yamamoto commented:
作成分3
[![Image from Gyazo](https://i.gyazo.com/0d51fc11fdd54ac9a746803f3aff89b7.jpg)](https://gyazo.com/0d51fc11fdd54ac9a746803f3aff89b7)

Ken-Yamamoto commented:
作成分4
[![Image from Gyazo](https://i.gyazo.com/2bdfd960b27fb679e29534ce79e4370d.jpg)](https://gyazo.com/2bdfd960b27fb679e29534ce79e4370d)

miyagit commented on app/assets/stylesheets/userRegistration.scss:
```diff
@@ -0,0 +1,358 @@
+                color: #fff;
+                font-size: 12px;
+                vertical-align: top;
+              }
```
ここインデント崩れてますね。。

Ken-Yamamoto replied:
修正しました。

miyagit commented on app/views/products/userRegistration.html.haml:
```diff
@@ -0,0 +1,124 @@
+              %label.inner__address__prefecture__head 都道府県
+              %span.inner__address__prefecture__require 任意
+              %select.inner__address__prefecture__form{:name => "pref"}
+                %option{:selected => "selected", :value => "--"} --
```
こちらactivehash使ったほうが良いですね。

Ken-Yamamoto replied:
activehash使用にて変更しました。

Ken-Yamamoto committed:
Modify on indents & pull down of 都道府県

下記編集および追加
userRegistration.html.haml active_hash使用に伴う変更
userRegistration.scss
app/models/prefecture.rb ファイル追加　都道府県プルダウンをactive_hashで実装
Gemfile  gem 'active_hash’追加
Gemfile.lock 上記に伴う追加

Ken-Yamamoto committed:
Modify  indents

TOMARUMARU reviewed (approved):
LGTM
